tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
+    "lib": ["es2016", "dom"],
     "rootDir": ".",
-    "outDir": "build"
```

Comment:
these new rules were added during @alexander-fenster's generation of the client, and I believe are automatic.